packages/core/src/browser/style/sidepanel.css

```diff
@@ -171,7 +171,42 @@
     color: var(--theia-sideBar-foreground);
 }
 
+.theia-app-sidebar-container {
+  min-width: var(--theia-private-sidebar-tab-width);
+  max-width: var(--theia-private-sidebar-tab-width);
+  background: var(--theia-activityBar-background);
+  display: flex;
+  flex-direction: column;
+}
 
+.theia-app-sidebar-container .theia-app-sides {
+  flex-grow: 1;
+}
+
+.theia-app-sidebar-container .theia-sidebar-bottom-menu {
+  flex-shrink: 0;
+}
+
+.p-Widget.theia-sidebar-bottom-menu {
+  background-color: var(--theia-activityBar-background);
+  display: flex;
+  flex-direction: column-reverse;
+}
+
+.p-Widget.theia-sidebar-bottom-menu i {
+  padding: 11px 10px;
```

Comment:
please use `var(--theia-ui-padding)` or calc based on it

Reply:
![image](https://user-images.githubusercontent.com/5847142/91289143-7faef480-e7c4-11ea-80b1-c9d572442371.png)
Maybe we can add a var(--theia-sidebar-tab-padding-top-and-bottom) and var(--theia-sidebar-tab-padding-left-and-right)?

Reply:
that's fine, but it is important that they should be dependent, changing of ui padding should be reflected everywhere